README.md

```diff
@@ -70,6 +70,7 @@
 ## 🚀 Live Demo <a name="live-demo"></a>
 
 - [Live Demo Link](https://hello-microverse.netlify.app/)
+- [Presentation video](https://www.loom.com/share/b2f3e5e5953d4757bb549d8f5228b566)
```

Comment:
- [OPTIONAL ] Nice work on making some changes from the previous review! Just a friendly reminder, You forget to mention the code pieces you're proud of. As your first Capstone Project, I'm sure you learned a lot and have code you're proud of. Share it with us. :wink: